bundles/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/handler/LGWebOSTVSocket.java

```diff
@@ -442,7 +442,11 @@ public boolean isConnected() {
 
     public ServiceSubscription<Float> subscribeVolume(ResponseListener<Float> listener) {
         ServiceSubscription<Float> request = new ServiceSubscription<>(VOLUME, null,
-                jsonObj -> "mastervolume_tv_speaker".equals(jsonObj.get("scenario").getAsString())
+                // "scenario" in the response determines whether "volume" is absolute or a delta value.
+                // it only makes sense to subscribe to changes in absolute volume
+                // accept: "mastervolume_tv_speaker" or "mastervolume_tv_speaker_ext"
+                // ignore external amp/receiver: "mastervolume_ext_speaker_arc" or "mastervolume_ext_speaker_urcu_oss"
+                jsonObj -> jsonObj.get("scenario").getAsString().startsWith("mastervolume_tv_speaker")
```

Comment:
Is jsonObj.get("scenario").getAsString() always guaranteed to return a String or can it also return null?

Reply:
In the protocol for volume subscriptions "scenario" is always present and filled with a value